docs/components/COMPONENT_REPLACEMENT.md

```diff
@@ -132,7 +132,7 @@ This configuration key's value is used at runtime to lookup in the component rep
 
 ## How components are made to accept replacement
```

Comment:
line 47 since I cannot put comments there: 
However, it allows the customization app to replace this empty functions with functions which provides the setup for custom components.
=> these empty functions